timemachine/cpp/src/bd_exchange_move.hpp

```diff
@@ -48,13 +51,14 @@ template <typename RealType> class BDExchangeMove : public Mover {
     DeviceBuffer<size_t> d_num_accepted_;
     DeviceBuffer<int> d_target_mol_atoms_;
     DeviceBuffer<int> d_target_mol_offsets_;
+    DeviceBuffer<__int128> d_intermediate_sample_weights_; // ceil_divide(N_, DEFAULT_THREADS_PER_BLOCK)
```

Comment:
Comment is stale, need to fix

Reply:
Removed in https://github.com/proteneer/timemachine/pull/1225/commits/a95fc2e41f56d3b2bb727b1e14e99a568fc0c1e9